pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "volttron-lib-tagging"
+version = "0.1.0-rc"
+description = "A base tagging agent api with extension points for using with the VOLTTRON platform. This base agent works with Haystack 3 tags"
+authors = ["VOLTTRON Team <volttron@pnnl.gov>"]
+license = "Apache License 2.0"
+readme = "README.md"
+repository = "https://github.com/eclipse-volttron/volttron-lib-tagging"
+homepage = "https://github.com/eclipse-volttron/volttron-lib-tagging"
+keywords = []
+packages = [ { include = "tagging", from = "src" }, { include = "tagging", from = "tests" } ]
```

Comment:
I don't like having the tests be a package...here...unless that is for including testing down stream.

Reply:
integration tests for implementing tagging agent such as sqlite-tagging will depend on the base tagging integration test interface. So if it is include in the whl it would be easier to develop and test concrete tagging agents